setup.py

```diff
@@ -12,7 +12,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/DMSC-Instrument-Data/dataset_viewer",
-    install_requires=["xarray","pyqt5","pyqt5-sip","netcdf4"],
+    install_requires=["xarray","pyqt5-sip","pyqt5","netcdf4", "matplotlib"],
```

Comment:
Unfortunately, i've been having problems with the default `easy_install` for setup.py when it comes to pqyt5. 

I can ONLY `python setup.py install` correctly if i have first `pip install pqyt5-sip pyqt5` first. I have seen suggestion on slack overflow that it's possible to make the setup.py use pip instead.